README.md

```diff
@@ -214,5 +228,73 @@ Running the above obfuscation on the string `a wonderful evening to go dancing`
 `xxx.xxx.xxx.xxx wonderful evening to go dxxx.xxx.xxx.xxxncing`, which might be counter-intuitive.
 So what happened here? First off, we would replace all `a` with a `b`, that `b` in turn would be replaced with `192.168.2.1` that later matches as an IPv4 and gets obfuscated in a static manner. 
 
-You can ensure that this does not happen, by providing custom obfuscators at the very bottom of the definition, preferably after all built-ins, and by ensuring you match on very specific terms (eg by supplying word boundaries in regular expressions).
+You can ensure that this does not happen, by providing custom obfuscators at the very bottom of the definition, preferably after all built-ins, and by ensuring you match on very specific terms (for example by supplying word boundaries in regular expressions).
+
+## Omission
+
+To ensure certain files will never be shared, must-gather-clean helps you to omit files. 
+There is support for the most common types of files in the must-gather:
+* [File Pattern](#file-pattern)
+* [Kubernetes Resource](#kubernetes-resource)
+
+### File Pattern 
+
+File patterns can be useful to omit certain folders and files by their path. The paths are always relative to the root of the must-gather that was supplied by the "-i" flag.
+
+```
+config:
+  omit:
+  - type: File
+    pattern: "*/namespaces/openshift-sdn/pods/*/*/*/logs/*.log"
+```
+
+This example illustrates how the globbing of the path works, you need to supply the respective folder structure down to the specific files you want to omit. A simple `*.log` will not suffice to omit all files matching the log extension in the must-gather. 
+The rationale here is to be explicit about what is being omitted to later avoid chasing the accidentally missing files.
+
+For more information about the supported syntax refer to the documentation on [filepath.Match](https://pkg.go.dev/path/filepath#Match) that powers this feature.
+
+### Kubernetes Resource
+
+Most of the very confidential information, for example authentication tokens and certificates, are stored in Kubernetes resources and its yaml representation. We added the ability to omit those resources by their familiar Kubernetes resource identifiers. 
+
+You can configure to omit all secrets by its kind as follows:
+
+```
+config:
+  omit:
+  - type: Kubernetes
+    kubernetesResource:
+       kind: "Secret"
+```
+
+which will, irrespectively of its API version and namespace, omit all Secrets that are defined in the must-gather. `kind` must always be supplied as a mandatory parameter.
+
+To supply one or many namespaces, you can supply a list as follows:
+
+```
+config:
+  omit:
+  - type: Kubernetes
+    kubernetesResource:
+       kind: "Secret"
+       namespaces: ["kube-system", "default"]
+```
+
+which will only omit the secrets in the defined namespaces.
```

Comment:
If I understand the technicality correctly, doesn't it mean this?
```suggestion
which will omit only the secrets in the defined namespaces.
```
You can correct if my understanding is right.

Reply:
absolutely correct, let me commit this